### DISCUSSION TITLE
The TTS model file

### DISCUSSION THREAD

**Discussion by Biflez48** [Camb-ai/MARS5-TTS#60] [Q&A] [answered]
After training the TTS model, where is the model stored? Is it saved to disk or in RAM? Can the trained model be reused without having to train it again?

**Comment by pieterscholtz** [comment, marked as answer]
@Biflez48 I'm not sure I understand the question. We have not provided the training scripts for MARS5. When running the demo the model checkpoints will be downloaded automatically and cached locally so that each time subsequent time you run inference, the cached model will be used.
